manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -228,6 +228,8 @@ data:
     - '{__name__="che_workspace_start_time_seconds_sum"}'
     # (codeready workspaces, @ibuziuk) The overall number of attempts for starting all the workspaces.
     - '{__name__="che_workspace_start_time_seconds_count"}'
+    # (cloud credential operator, @openshift/openshift-team-hive) Track current mode the cloud-credentials-operator is functioning under.
```

Comment:
Is this operator installed by default or is an optional component?

Reply:
Installed by default CVO operator.